changelogs/fragments/1912-yum_versionlock-lock_unlock_concurrently.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - yum_versionlock - Do the lock/unlock concurrently to speed up (https://github.com/ansible-collections/community.general/pull/1912).
```

Comment:
As this PR is more than just fixing #1908 and the issue is about disabling tests I decided to put PR link here, please tell me if this needs to be changed.

Reply:
I think it's fine as-is!